benchmarks/viscoelastic_plastic_shear_bands/gerya_2019/gerya_2019_analysis.py

```diff
@@ -0,0 +1,100 @@
+# Python script to load the solution data into a
+# numpy array, interpolate the data to a new
+# uniform grid, and plot the results.
+# Within a python interpreter, the script can 
```

Comment:
This is in ParaView or a standalone python code? Why don't you say ``python gerya_2019_analysis.py``?

Reply:
That would work as well. The command I listed is for when you want to execute the script while working inside ipython (or similar). I find this helpful for doing quick debugging and testing without needing to rerun the whole script. In any case, good to spell out both options, which is now included in the documentation. In the last commit I also specified the versions of each package in the documentation.